README.md

```diff
@@ -79,21 +73,41 @@ do the following:
 
 ### Developing with Docker Compose
 
-This will explain how to start the server and its dependencies usin Docker, create AWS/OCP sources, and view reports. This will not cove all API or scenarios but should give you an end to end flow.
+This will explain how to start the server and its dependencies usin Docker, create AWS/OCP sources, and view reports. This will not cover all API or scenarios but should give you an end-to-end flow.
 
 #### Starting Koku using Docker Compose
 
 > **Note**
 >
 > In order for the `koku_base` image to build correctly, buildkit must be enabled by setting `DOCKER_BUILDKIT=1`. This is set in the `.env` file, but if you are having issues building the `koku_base` image, make sure buildkit is enabled.
 
-1.  Start the docker containers:
+1.  Start the containers:
+
+        make docker-up-min-trino
+
+2.  Display log output from the containers. It is recommended that logs be kept in a second terminal:
+
+        make docker-logs
+
+With all containers running any source added will be processed by saving CSV files in MinIO and storing Parquet files in MinIO. The source's data will be summarized via Trino. Summarized data will land in the appropriate daily_summary table for the source type for consumption by the API.
+
+To add test sources and data:
+
+    make create-test-customer
+    make load-test-customer-data (optional)start={start_date} (optional)end={end_date}
```

Comment:
Could also specify a test_source, e.g. `test_source=AWS`

Reply:
Great idea. I use that quite a bit.